rasa/core/processor.py

```diff
@@ -105,9 +105,15 @@ def _load_model(
         model_path: Union[Text, Path]
     ) -> Tuple[Text, ModelMetadata, GraphRunner]:
         """Unpacks a model from a given path using the graph model loader."""
-        model_tar = get_latest_model(model_path)
-        if not model_tar:
-            raise ModelNotFound(f"No model found at path '{model_path}'.")
+        try:
+            if os.path.isfile(model_path):
+                model_tar = model_path
```

Comment:
Does this change require adding / updating a test?

Reply:
the new test `test_loads_correct_model_from_path` breaks before this is added.